release.py

```diff
@@ -0,0 +1,488 @@
+except ImportError:
+    from distutils.core import Command
+
+GITHUB_REPOSITORY_API = 'https://api.github.com/repos/prestodb/presto-admin'
+CURRENT_DIRECTORY = os.path.dirname(os.path.realpath(__file__))
+
+
+class SemanticVersionHelper(object):
+    def __init__(self, version):
+        self.version = version
+        self._set_semantic_version_fields()
+
+    @staticmethod
+    def _get_version_field_value(version_fields, index):
+        try:
+            return version_fields[index]
+        except IndexError:
+            # The field value was omitted for the version
+            return 0
+
+    def _set_semantic_version_fields(self):
+        version_fields = self.version.split('.')
+        self.major_version = self._get_version_field_value(version_fields, 0)
+        self.minor_version = self._get_version_field_value(version_fields, 1)
+        self.patch_version = self._get_version_field_value(version_fields, 2)
+
+    def __lt__(self, other):
+        if self.major_version == other.major_version:
+            if self.minor_version == other.minor_version:
+                return self.patch_version < other.patch_version
+            else:
+                return self.minor_version < other.minor_version
+        else:
+            return self.major_version < other.major_version
+
+    def __str__(self):
+        return self.version
+
+    @staticmethod
+    def _bump_version(version_field):
+        return str(int(version_field) + 1)
+
+    def _get_acceptable_major_version_bumps(self):
+        acceptable_major = self._bump_version(self.major_version)
+        return [acceptable_major,
+                acceptable_major + '.0',
+                acceptable_major + '.0.0']
+
+    def _get_acceptable_minor_version_bumps(self):
+        acceptable_minor = self._bump_version(self.minor_version)
+        return [str(self.major_version) + '.' + acceptable_minor,
+                str(self.major_version) + '.' + acceptable_minor + '.0']
+
+    def _get_acceptable_patch_version_bumps(self):
+        acceptable_patch = self._bump_version(self.patch_version)
+        return [str(self.major_version) + '.' + str(self.minor_version) + '.' + acceptable_patch]
+
+    def get_acceptable_version_bumps(self):
+        """
+        This functions takes as input strings major, minor, and patch which should be
+        the corresponding semvar fields for a release. It returns a list of strings, which
+        contains all acceptable versions. For each field bump, lower fields may be omitted
+        or 0s. For instance, bumping 0.1.2's major version can result in 1, 1.0, or 1.0.0.
+        """
+        major_bumps = self._get_acceptable_major_version_bumps()
+        minor_bumps = self._get_acceptable_minor_version_bumps()
+        patch_bumps = self._get_acceptable_patch_version_bumps()
+        return major_bumps + minor_bumps + patch_bumps
+
+
+class ReleaseFetcher:
+    def __init__(self, directory, github_api_directory):
```

Comment:
suggest: github_api_path; it isn't really a directory in the sense that it's a container for files.